rules/aep0004/resource_singular.go

```diff
@@ -32,7 +32,7 @@ var resourceSingular = &lint.MessageRule{
 		l := locations.MessageResource(m)
 		s := r.GetSingular()
 		_, typeName, ok := utils.SplitResourceTypeName(r.GetType())
-		lowerTypeName := utils.ToLowerCamelCase(typeName)
+		lowerTypeName := utils.ToKebobCase(typeName)
```

Comment:
```suggestion
		lowerTypeName := utils.ToKebabCase(typeName)
```
https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case - small typo I think.